src/actions/authActions.js

```diff
@@ -144,7 +136,7 @@ export const loginAction = (
 
       if (pin) {
         const saltedPin = await getSaltedPin(pin, dispatch);
-        const decryptionOptions = generateNewConnKeys ? { mnemonic: true } : {};
+        const decryptionOptions = {};
         wallet = await decryptWallet(encryptedWallet, saltedPin, decryptionOptions);
```

Comment:
Options of `decryptWallet` is optional param, right?
```suggestion
        wallet = await decryptWallet(encryptedWallet, saltedPin);
```

Reply:
According to spec - yes :)